src/tailwind.css

```diff
@@ -67,4 +67,16 @@
   .form-check-input:checked[type="radio"] {
     background-image: url("data:image/svg+xml,%3Csvg xmlns=%22http://www.w3.org/2000/svg%22 viewBox=%22-4 -4 8 8%22%3E%3Ccircle r=%222%22 fill=%22%23fff%22/%3E%3C/svg%3E");
   }
+
+  /**
+   * Dropdowns.
+   */
+
+  .dropdown-menu {
+    @apply flex flex-col rounded-md  bg-stone-600 drop-shadow-md;
+  }
```

Comment:
I think this is simple enough that we're better off just using the class names in the `Dropdown` component directly, instead of needing to keep it in a separate file.

Reply:
changed in latest commit